pyproject.toml

```diff
@@ -1,2 +1,91 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "sagemaker"
+dynamic = ["version", "optional-dependencies"]
+description = "Open source library for training and deploying models on Amazon SageMaker."
+readme = "README.rst"
+license = "Apache-2.0"
```

Comment:
Just saw this but can you change this to be `license = {file = "LICENSE.txt"}` like mentioned [here](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license)

Reply:
That is outdated actually the standard is being updated by [PEP 639](https://peps.python.org/pep-0639/) which was always planned (allowing a string value). The license file is automatically picked up by Hatchling and included in the source distribution.

Reply:
The only blocker I have with this one currently is that when I run `python setup.py sdist` locally I get an error like:

```
configuration error: `project.license` must be valid exactly by one definition (2 matches found):

    - keys:
        'file': {type: string}
      required: ['file']
    - keys:
        'text': {type: string}
      required: ['text']

DESCRIPTION:
    `Project license <https://peps.python.org/pep-0621/#license>`_.

GIVEN VALUE:
    "Apache-2.0"

OFFENDING RULE: 'oneOf'
```

Reply:
Because [PEP 639](https://peps.python.org/pep-0639/) is still in Draft can we do `license = {file = "LICENSE.txt"}` until then?

Reply:
Done! I removed the field instead because the license is automatically added to the source distribution.